lib/sles4sap_publiccloud.pm

```diff
@@ -373,13 +402,20 @@ sub check_takeover {
 }
 
 =head2 enable_replication
-    enable_replication();
+    enable_replication([site_name => 'site_a']);
 
     Enables replication on fenced database. Database needs to be offline.
+
+=over 1
+
+=item B<SITE_NAME> - site name of the site to register
+
+=back
 =cut
 
 sub enable_replication {
-    my ($self, $site_name) = @_;
+    my ($self, %args) = @_;
```

Comment:
Function takes only one argument which is described above. Is there a reason to ask for argument name specifically?

Reply:
My personal preference here is always to use the same approach. I prefer `%args` as:
- argument are "named" in the prototype, even the caller has named arguments
- I find more easy to validate them
- I find more prominent within the sub implementation what is an argument variable and what is a local variable